manifests/init.pp

```diff
@@ -261,6 +264,7 @@
   Stdlib::Absolutepath $rdeck_home                              = $rundeck::params::rdeck_home,
   Boolean $manage_home                                          = $rundeck::params::manage_home,
   Optional[String] $rdeck_profile_template                      = undef,
+  Optional[String] $rdeck_override_template                     = $rundeck::params::override_template,
```

Comment:
since it's always `rundeck/profile_overrides.erb` you can also directly assign the string to the variable and don't go the route via params.pp. Also If the default is not `undef` you don't need to set the type to `Optional[]` because you cannot set the variable to `undef`. If you pass `undef` to it it will always be overwritten by  `rundeck/profile_overrides.erb` . That's probably not the behaviour you intended?

Reply:
Right, fixing the optional clause on the next commit. About the params, all parameters specified in this module are passed through params.pp

Reply:
@josecastroleon we're trying to get away from the `params.pp` pattern, and have defaults directly in `init.pp` because Puppet Strings can make nicer documentation that way.